### PR TITLE
Update NodeJS version from 10.15.1 to 10.23.0

### PR DESCRIPTION
**Pull request by lfdebrux**
We're running Node 10.23.0 in production[[1]].

[1]: https://github.com/alphagov/digitalmarketplace-docker-base/blob/138fadae2285d9631a41d06db439113a9acd3714/frontend.docker#L3

### CODE REVIEW

File: package/package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   "main": "digitalmarketplace/all.js",
   "sass": "digitalmarketplace/all.scss",
   "engines": {
-    "node": "10.19.0"
+    "node": ">= 4.2.0"
```
Comment (gidsg) [inline package/package.json:11]: do we really support any node version from 4.2.0 upwards?

Comment (lfdebrux) [reply package/package.json:11]: Hmmmm. Just tried running `npm install` with node 4.2.0 and got a truly amazing number of warnings.

Maybe `>= 10.19.0` would be more realistic?

Comment (gidsg) [reply package/package.json:11]: @lfdebrux I would guess anything 10.0+ has the same API.

**Review by gidsg — APPROVED** [review]
🆙 👍

**Pull request merged**